docs/releases/1.10.txt

```diff
@@ -189,6 +189,11 @@ Requests and Responses
   :meth:`~django.http.HttpResponse.seekable()` to make an instance a
   stream-like object and allow wrapping it with :py:class:`io.TextIOWrapper`.
 
+* Iterators you pass to :class:`~django.http.HttpResponse`
```

Comment:
It's unclear to me why this is considered a "minor feature". Seems more like a bug fix to me.